src/config.ts

```diff
@@ -500,7 +500,7 @@ export default {
    */
   STALL_GAP: {
     DEFAULT: 0.5,
-    LOW_LATENCY: 0.001,
+    LOW_LATENCY: 0.2,
```

Comment:
:O Why? Don't we risk to rebuffer much more often here?

Reply:
Maybe we should find a compromise, but if the value is too low (0.001), the content is __not__ considered as buffering, as there is almost always more than 0.001 of buffer ahead of current position.